GUI/src/cz/fidentis/gui/actions/SaveProject.java

```diff
@@ -613,6 +613,50 @@ private Element addBatchComparison(Element root, ProjectTopComponent tc) throws
         comparisonE.setAttribute("icpNumOfHeads", String.valueOf(comparison.getICPnumberOfHeads()));
 
         comparisonE.setAttribute("templateIndex", String.valueOf(comparison.getTemplateIndex()));
+        
+        comparisonE.setAttribute("continueComparison", String.valueOf(comparison.isContinueComparison()));
+        
+        comparisonE.setAttribute("firstCreated", String.valueOf(comparison.isFirstCreated()));
+        
+        comparisonE.setAttribute("visualization", String.valueOf(comparison.getVisualization()));
+        
+        comparisonE.setAttribute("crossCutPlaneIndex", String.valueOf(comparison.getCrossCutPlaneIndex()));
```

Comment:
Nedali by sa niektore tieto veci nejako zlucit do nejakej logicky suvisiacej casti a spravit z toho napr. samostatny element? Zjednodusil by sa kod a mozno by sa to dalo aj znovupouzit pre vsetky projekty. (Podobne je myslim robeny ten HDPaint ci jak sa to vola)
Napadlo ma to lebo tieto veci vypadaju ako niekolko roznych premennych ktore ale logicky patria k sebe lebo ovladaju nejaku vizualizaciu je to tak?